ecosystem/sep-0010.md

```diff
@@ -3,10 +3,10 @@
 ```
 SEP: 0010
 Title: Stellar Web Authentication
-Author: Sergey Nebolsin <sergey@mobius.network>, Tom Quisel <tom.quisel@gmail.com>, Leigh McCulloch <@leighmcculloch>
+Author: Sergey Nebolsin <sergey@mobius.network>, Tom Quisel <tom.quisel@gmail.com>, Leigh McCulloch <@leighmcculloch>, Jake Urban <jake@stellar.org>
 Status: Active
 Created: 2018-07-31
-Updated: 2020-04-03
+Updated: 2020-08-14
 Version 1.3.1
```

Comment:
I think we need to bump the version, since this introduces a breaking change to the protocol.

Reply:
I'm unfamiliar with the versioning scheme. Would this constitute a patch, minor, or major version bump?

Reply:
Agreed we should bump this to 2.0, since clients who implement this will reject outdated server-side SEP-10 transactions.